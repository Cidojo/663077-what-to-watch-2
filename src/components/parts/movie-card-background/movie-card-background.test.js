import * as React from 'react';
import {MovieCardBackground} from './movie-card-background.jsx';
import * as renderer from 'react-test-renderer';

const props = {
  name: ``,
  backgroundImage: ``,
};

it(`should render MovieCardBackground component without errors`, () => {
  const tree = renderer.create(<MovieCardBackground {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
