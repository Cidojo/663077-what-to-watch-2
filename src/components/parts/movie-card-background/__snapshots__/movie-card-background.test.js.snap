// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MovieCardBackground component without errors 1`] = `
<div
  className="movie-card__bg"
>
  <img
    alt=""
    src=""
  />
</div>
`;
